src/web/browser/coreVitals/init.ts

```diff
@@ -5,7 +5,8 @@ import { coreVitals } from './coreVitals';
 const init = async (): Promise<void> => {
 	// Sample 1 out of 100
 	const inSample = Math.floor(Math.random() * 100);
-	if (inSample === 1) {
+	const inLocal = window.location.hostname === 'localhost';
+	if (inSample === 1 || inLocal) {
```

Comment:
Worth noting that the name of this var is slightly misleading, as `inSample === 20` would have the same effect here.

I think it’s easier to maintain something like this:

```
const inSample = Math.random() <= 1/100;
```